docs/guide/migration/index.md

```diff
@@ -67,6 +67,10 @@ createRouter({
 })
 ```
 
+### Removal of `fallback` option
+
+The `fallback` option has been removed from `RouterOptions`. In the cases that the browser does not support `history.pushState` or `history.replaceState`, the router will use `location.replace` or `replace.assign` for navigation. On SSR, you need to set the `history` option of router as [createMemoryHistory](https://next.router.vuejs.org/api/#creatememoryhistory).
```

Comment:
```suggestion
The `fallback` option is no longer supported when creating the router:

```diff
- new VueRouter({
+ createRouter({
-   fallback: false,
 // other options...
 })

**Reason**: All browser supported by Vue support the [HTML5 History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API), allowing us to avoid hacks around modifying `location.hash` and directly use `history.pushState()`.
```

Reply:
hmm, this suggestion is not working

Reply:
> hmm, this suggestion is not working

So should I change my description to be this one :
```shell
The `fallback` option is no longer supported when creating the router:
```diff
-new VueRouter({
+createRouter({
-  fallback: false,
// other options...
})
```